### DISCUSSION TITLE
official "Simple Text Label" example fails, invalid argument for option '-geometry'

### DISCUSSION THREAD

**Discussion by iambumblehead** [ImageMagick/ImageMagick#7144] [Help]
### ImageMagick version

7.1.1-29

### Operating system, version and so on

Arch Linux, x86

### Description

The official ["Simple Label Text" example][0] call fails on this system,
```bash
magick -background lightblue -fill blue \
      -font Cantarell -pointsize 72 label:Anthony \
      label.gif
convert: invalid argument for option '-geometry':  @ error/convert.c/ConvertImageCommand/1757.
```

I've tried *many* different variations of this call with no success. Any help or advice is appreciated. Thank you.

[0]: https://www.imagemagick.org/Usage/text/#label
[1]: https://github.com/sdushantha/fontpreview/blob/master/fontpreview#L56

**Comment by snibgo** [comment]
I can't reproduce the problem with IM v7.1.1-28 on Windows 11.

I don't know why IM is complaining about `-geometry` when that setting isn't in the command.

Please run your command again, with `-debug all`, and paste the text output here.
```
magick -debug all -background lightblue -fill blue \
      -font Cantarell -pointsize 72 label:Anthony \
      label.gif
```

**Comment by iambumblehead** [comment]
how strange! the command now works without any error! thanks for responding to me :)
